news/2018-03-06-new-featured-artist-trial-and-error.md

```diff
@@ -0,0 +1,68 @@
+
+But don't take our word for it, have a listen to a small sample of what's available and see for yourself!
+
+### Yoru wo Kakeru feat. ming-zi
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/383.mp3" type="audio/mpeg">
+</audio>
+
+> An airy vocal walk featuring the talent of ming-zi, Yoru wo Kakeru levels traditional instrumentation together with sweeping lyrics. Nothing particularly new, but an excellent take on a more standard musical approach.
+
+### DAI★TAN SENSATION feat. Nanahira, Mii, Nakuru Aitsuki
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/381.mp3" type="audio/mpeg">
+</audio>
+
+> Uplifting and with all the sugar and sweetness expected from an anime-styled track, DAI★TAN SENSATION is a playful romp among the genre, with the added bonus of the vocals from a few prominent singers among the Japanese music scene!
+
+### Hoshizora Touhikou feat. Hatsune Miku
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/372.mp3" type="audio/mpeg">
+</audio>
+
+> A heartfelt ballad rendered in the voice of the Vocaloid Hatsune Miku, Hoshizora Touhikou is a slow and paced walk throughout an sombre starscape, tied off with gentle, leading percussion towards the later parts of the track.
+
+### Tokoyami no Keiyaku KEGARETA-SHOUJO feat. GUMI
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/377.mp3" type="audio/mpeg">
+</audio>
+
+> Beset with a backing choir, the voice of the Vocaloid GUMI reigns supreme in this fast-paced, rock-themed piece.
+> 
+
+### Houkai no Hate No Ishi feat. GUMI
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/20/previews/373.mp3" type="audio/mpeg">
+</audio>
+
+> Slower paced and more pensive, Houkai no Hate No Ishi is a faintly ethereal take on a classic rock sound, set to the voice of the Vocaloid GUMI.
+
+------
+
+But that's not all, there's a few more tracks on [**Trial & Error**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/20) for you to check out if you're interested. 
+
+Additionally, every track comes with a "short version" cut around roughly 1m30s in length - perfectly poised for mappers to get stuck in and experimenting with his sound.
+
+If you like what you hear, make sure to check out his [official website](https://tandess.itch.io/) for more tracks that you might like, [his twitter](http://twitter.com/tandess) for all the details on new releases, and maybe even consider subscribing to his [Patreon](https://www.patreon.com/tandess) as well.
+
+We're also proud to announce that a number of existing Featured Artist listings will receive some brand new track updates in the coming weeks, bringing some new variety to the artists that you know and love. These will be announced via both newspost and Twitter as is the normal, so make sure not to miss out!
```

Comment:
Featured Artist listings -> Featured Artists / Featured Artist pages?

does it mean that there will be new artists or new tracks on artists' pages?